package.json

```diff
@@ -55,7 +55,7 @@
     "continuation-local-storage": "3.x",
     "dateformat": "^3.0.3",
     "find-root": "^1.1.0",
-    "lodash": "=4.17.9",
+    "lodash": "^4.17.9",
```

Comment:
what was the reason to fix this version?

Reply:
`wd` used to patch `lodash` with `underscore`'s string utilities, for reasons that made sense in 2014. If Appium had the same version of `lodash` then `_.truncate` would just return `...` no matter what the input. The "fix", before anyone could figure out what the actual problem was, was to pin `lodash` (when it was pinned, it seemed like probably a `lodash` issue, with 4.17.10). 

So, `wd` is fixed now, so this doesn't need to be pinned!